exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -283,7 +282,7 @@ func (prwe *PrwExporter) export(ctx context.Context, tsMap map[string]*prompb.Ti
 	var mu sync.Mutex
 	var wg sync.WaitGroup
 
-	concurrencyLimit := int(math.Min(maxConcurrentRequests, float64(len(requests))))
+	concurrencyLimit := int(math.Min(float64(prwe.minShards), float64(len(requests))))
```

Comment:
If `requests` get a value less than `minShards`, then concurrencyLimit will indeed be less than such `minShards` value, no?

Reply:
True, this is slightly different than how Prom always have up to min_shards workers. See my other comment at https://github.com/open-telemetry/opentelemetry-collector/pull/3046#discussion_r633721031. I'm happy to make this looking more similar to Prometheus.